sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessor.java

```diff
@@ -98,9 +107,12 @@ public void onEnd(ReadableSpan span) {
         result.whenComplete(
             () -> {
               pendingExports.remove(result);
+              String error = null;
               if (!result.isSuccess()) {
                 logger.log(Level.FINE, "Exporter failed");
+                error = "export_failed";
```

Comment:
#nit: for consistency 
```suggestion
                if (result.getFailureThrowable() != null) {
                  error = result.getFailureThrowable().getClass().getName();
                } else {
                  error = "export_failed";
                }
```